package.json

```diff
@@ -166,6 +166,7 @@
       "dot-notation": "error",
       "func-name-matching": "error",
       "guard-for-in": "error",
+      "no-confusing-arrow": "error",
```

Comment:
Needs to be updated to avoid conflicts with Prettier: https://github.com/prettier/eslint-config-prettier#no-confusing-arrow

Reply:
Ha, thanks, I thought this rule wasn't in the Prettier list 😏

Fixed in https://github.com/stylelint/stylelint/pull/4514/commits/0c7ea8da21bc74530384186ea62075c92e7fa7df